pkg/presets/presets.go

```diff
@@ -17,32 +17,41 @@ var (
 	PrincipalMapping   map[string]string
 )
 
-// Setting this value to anything will cause production presets to be used, else use warm staging.
-var ProductionPresetsEnvVar = "PIRI_PRODUCTION_PRESETS"
```

Comment:
Changed this to `PIRI_PRESETS` so that we don't have to have a different env var for each deployment. Technically a breaking change but actually it didn't work if you set it as there was no `else` or `return` on the conditional so the prod values got overwrittemn by warm staging values.